src/m365/teams/commands/app/app-list.ts

```diff
@@ -47,100 +42,38 @@ class TeamsAppListCommand extends GraphCommand {
   #initTelemetry(): void {
     this.telemetry.push((args: CommandArgs) => {
       Object.assign(this.telemetryProperties, {
-        all: args.options.all || false,
-        teamId: typeof args.options.teamId !== 'undefined',
-        teamName: typeof args.options.teamName !== 'undefined'
+        distributionMethod: typeof args.options.distributionMethod !== 'undefined'
```

Comment:
Since this option is a choice field, let's just log the option value.

```suggestion
        distributionMethod: args.options.distributionMethod || false
```